README.md

```diff
@@ -90,6 +96,41 @@ To accept inbound connections, add the `--listen` flag.
 $ hsd --listen --max-inbound=20
 ```
 
+### Account and Wallet Setup
+HSD uses a hierarchical deterministic wallet structure. The hsw-cli binary is used to interact with the HSD wallet.
+
+Make sure you have a HSD node running.
+
+To view the extended public key and mnemonic details (entropy and phrase information) by:
+
+`$ hsw-cli master`
+
+To view the `default` account details, such as the `accountKey`, by:
+
+` $ hsw-cli account get default`
+
+To make a new wallet in the `default` account, use the `mkwallet` command and provide a wallet id. It is custom, but not required, to name the first wallet id `primary`.
+
+`$ hsw-cli mkwallet <WALLET_ID>`
+
+To view the wallet details such as the public key and wallet address use the `address` command and provide the wallet id and account name (`default`) previously defined:
+
+`$ hsw-cli address --id=<WALLET_ID> --account=<ACCOUNT_NAME>`
+
+To create a new account:
+
+`$ hsw-cli account create <NEW ACCOUNT NAME>`
+
+To view existing accounts:
+
+`$ hsw-cli account list`
+
+To view default account balance:
+
+`$ hsw-cli balance`
+
+To view the balance of a different account, add the `--account=<ACCOUNT_NAME>` option flag.
```

Comment:
It might be worth noting at the beginning of these commands that any of the account specific calls can be modified by the `--account` flag to access a different account.